internal/service/chat_service.go

```diff
@@ -387,3 +387,37 @@ func (s *ChatService) HasNextMessages(
 
 	return hasNext, nil
 }
+
+// 채팅 메시지를 저장합니다.
+func (s *ChatService) SaveChatMessage(
+	ctx context.Context, userID, roomID uuid.UUID, messageType, content string,
+) (*chat.Message, *pnd.AppError) {
+	chatMessageID, uuidError := uuid.NewV7()
+	if uuidError != nil {
+		return nil, pnd.ErrUnknown(fmt.Errorf("failed to generate UUID: %w", uuidError))
+	}
+
+	tx, transactionError := s.conn.BeginTx(ctx)
+	defer tx.Rollback()
+
+	if transactionError != nil {
+		return nil, pnd.FromPostgresError(transactionError.Err)
+	}
+
+	q := databasegen.New(tx)
+	row, databaseGenError := q.SaveChatMessage(ctx, databasegen.SaveChatMessageParams{
+		ID:          chatMessageID,
+		UserID:      userID,
+		RoomID:      roomID,
+		MessageType: messageType,
+		Content:     content,
+	})
+
+	if databaseGenError != nil {
+		return nil, pnd.FromPostgresError(databaseGenError)
```

Comment:
여긴 이렇게 바꿔주시면 될 것 같습니다
```suggestion
		return nil, databaseGenError
```